### PR TITLE
merge back ariadne branch with few fixes and update of scenario config

### PR DESCRIPTION
**Pull request by fschreyer**
This includes:

* some fixes of offshore wind and gas/coal power phase-out discussed with Robert
* new calibration
* new ariadne scenario config
* switch back H2 buildings and industry constraint to lower than as this caused issues with calibration
* extension of cm_regipol_slope_beforeTarget switch to region groups, e.g. EUR_regi including all EU subregions
* update ariadne plotting script

### CODE REVIEW

**Review by LaviniaBaumstark — COMMENTED** [review]
what about generisdata_tech_SSP1.prn and generisdata_tech_SSP5.prn? Should they also be adjusted?

**Comment by fschreyer** [comment]
Sry guys, for all the single commits. Please somebody approve if no more issues so that I can merge.

File: modules/47_regipol/regiCarbonPrice/declarations.gms
Comment (Renato-Rodrigues) [reply modules/47_regipol/regiCarbonPrice/declarations.gms:128]: the change is more generic than the first implementation allowing to create region targets for region groups (ex: EUR_regi) instead of only REMIND regions. as before

**Comment by Renato-Rodrigues** [comment]
I am still against the direct code changing of parameters, most specifically the pm_cf.
One thing is to do this while working on forks, but when merging to the trunk we shoudl avoid that, because this will just get in the limbo until some good soul decide to clean up the code.
It is also, most probably, an one line change into a single file in mrremind to define DEU specific capacity factors, so I see no reason to not move that to the proper place before the merge.

**Comment by fschreyer** [comment]
@Renato-Rodrigues 

Ok. I agree in principle. The thing is I dont know what the fate of these changes is. I can move the coal power CF change to mrremind, That makes sense. However, I dont know about the increased CF for spv and wind for Germany based on the ariadne modeling comparison and whether that should become default. That's why I hid it behind a switch. 

My suggestion would be I move this to mrremind as is and make it new default and I open an issue on validation of power sector CFs and hand over to @cchrisgong or @robertpietzcker or somebody else.

**Comment by fschreyer** [comment]
Ok, I removed the CF change of coal power in regipol/datainput for Germany and will move it to mrremind. That's all I can offer. Cleaning up the other pm_cf changes requires more work as
* pm_cf is 1 for all VRE technologies and just scaled up by this switch to align with ariadne values, it does not make sense to change this now in mrremind and would require a change in calcSolar and calcWindPotential which are more involved. This requires a general overhaul if necessary. For now, this is behind a switch but it should checked whether we need it everywhere. 
* The scaling over time of wind CFs that we have requires REMIND internal parameters (``p_aux_capacityFactorHistOverREMIND``) and therefore requires a more general approach if taken to mrremind. 

So, I will leave those changes but mention them in the issue. There, it should be clarified which short-term changes we need for coal/gas CFs and which general changes we need for VRE CFs.

**Comment by Renato-Rodrigues** [comment]
Sounds perfect for me. Sorry for being stubborn on this :)

**Review by Renato-Rodrigues — APPROVED** [review]

**Pull request merged**
